policy.md

```diff
@@ -1,178 +1,116 @@
-
-## 4. Intermediate Certificates ##
-
-An intermediate certificate is a certificate capable of issuing new
-certificates that is not a root certificate. To determine which intermediate 
-certificates must be entered into the CCADB, refer to the individual Store policy 
-documents. This includes certificates that are revoked. For
-newly-created intermediate certificates, this must happen before the
-certificate begins issuing publicly-trusted certificates.
-
-Each instance (i.e. PEM data) of an intermediate certificate only needs to be 
-included in the CCADB once, even if it chains up to two root certificates.
-
-If an intermediate certificate is revoked, the CCADB must be updated to mark it 
-as revoked, giving the reason why, within 24 hours for a security incident, and 
-within 7 days for any other reason.
-
-## 5. Policies, Practices and Audit Information ##
-
-In the records for both root certificates and intermediate certificates, CAs
-are required to provide details of the Certificate Policy (CP) and/or
-Certificate Practice Statement (CPS) documents relating to that certificate,
-and also statements of attestation of their conformance to various requirements
-and other operational criteria (“audits”), those statements being made by a
-competent independent party or parties. These documents are hosted elsewhere
-and the URLs are stored in the CCADB. The URLs to such CPs, CPSes and audits,
-and any metadata about them such as the name of the auditor or the date of the
-audit, need to be updated as new information become available. For technical
-reasons, URLs to audit statements need to point to a PDF file.
-
-The entry for each intermediate certificate has "Audits Same as Parent" and
-"CP/CPS Same as Parent" checkboxes. When those are checked, the details do not
-need to be duplicated from the parent cert. However, the intermediate
-certificate must be specifically listed in the audit statements of the parent
-certificate.
-
-URLs for the following documents are required for each certificate, 
-unless the exceptions below apply:
-
-* Certificate Policy (CP)
-* Certificate Practice Statement (CPS)
+A root CA certificate is a self-signed certificate issued by the root CA to identify itself and to facilitate verification of certificates issued to its subordinate CAs. CAs must maintain correct and current information about their root certificates.
+
+If root certificate information needs to be updated, the CA must submit an [Add/Update Root Request Case](https://docs.google.com/document/d/1ttmeeqO6WxDWe_deDNsGUgDO_LpsvoduFNZeHHMw_f8/edit).
+
+## 4. Subordinate Certificates ##
+
+An subordinate CA certificate is a certificate signed by a root CA or another subordinate CA. All subordinate CA certificates capable of chaining to a root CA certificate included in at least one participating Store must be added to CCADB within seven calendar days of issuance. This includes certificates that are revoked. 
```

Comment:
The CCADB SC should talk about this: "All subordinate CA certificates capable of chaining to a root CA certificate included in at least one participating Store must be added to CCADB within seven calendar days of issuance."

I think this may be a new requirement for CAs, so if all root store members agree to this new requirement, there should be a date by which this must be done.
For example for Mozilla we currently only require CAs to add the subordinate CA certificates that are relevant to our program.